src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,7 +14,12 @@
     private Phone phone;
     private Email email;
     private Address address;
-
+    
+    private static int NEXT_SEQUENCE_NUMBER_INITIAL_VALUE = 1;
```

Comment:
if this is a constant, i believe you need a `final` keyword also. but then again, this constant is a little trivial. If it is not reused else where, initializing the `nextSequenceNumber` with a constant would suffice.